server/serveUpdateFeed.js

```diff
@@ -1,31 +1,47 @@
 var semver = Npm.require('semver');
-
 var electronSettings = Meteor.settings.electron || {};
 var latestVersion = electronSettings.version;
 
-canServeUpdates = function() {
-  return latestVersion && DOWNLOAD_URL;
+canServeUpdates = function(platform) {
+  if (! latestVersion){
+    return false;
+  }
+  if (platform === "darwin"){
+    return typeof(DOWNLOAD_URL_OSX) !== 'undefined';
+  } else if (platform === "win32"){
+    return typeof(DOWNLOAD_URL_WIN32) !== 'undefined';
+  }
 };
 
 UPDATE_FEED_PATH = "/app/latest";
 
-// https://github.com/Squirrel/Squirrel.Mac#server-support
-serveUpdateFeed = function() {
-  // Abort if the user has not yet configured updates.
-  if (!canServeUpdates()) return;
 
-  serve(UPDATE_FEED_PATH, function(req, res, next) {
-    var appVersion = req.query.version;
+serveUpdateFeed = function() {
+  // https://github.com/Squirrel/Squirrel.Mac#server-support
+  if (canServeUpdates("darwin")){
+    serve(UPDATE_FEED_PATH, function(req, res, next) {
+      var appVersion = req.query.version;
+      if (semver.valid(appVersion) && semver.gte(appVersion, latestVersion)) {
+        res.statusCode = 204; // No content.
+        res.end();
+      } else {
+        res.statusCode = 200;
+        res.setHeader('Content-Type', 'application/json');
+        res.end(JSON.stringify({
+          url: DOWNLOAD_URL_OSX
+        }));
+      }
+    });
+  }
 
-    if (semver.valid(appVersion) && semver.gte(appVersion, latestVersion)) {
-      res.statusCode = 204; // No content.
+  // https://github.com/squirrel/squirrel.windows
+  if (canServeUpdates("win32")){
+    serveDir(UPDATE_FEED_PATH, function(req, res, next){
+      //first strip off the UPDATE_FEED_PATH
+      var path = req.url.split(UPDATE_FEED_PATH)[1];
+      res.statusCode = 302;
+      res.setHeader("Location", electronSettings.windowsDownloadURLPrefix + path);
```

Comment:
Can we rename the existing `downloadUrl` to make it clear that it's for Mac? We discussed a URL structure like [this](https://docs.google.com/document/d/1DxpGVp9CVZr9YdRljAuGW2BGiJImOoPltfLKWA25D7Q/edit#heading=h.52ypp5miwn6x) potentially